tests/snappi/pfcwd/files/pfcwd_multi_node_helper.py

```diff
@@ -143,6 +145,11 @@ def run_pfcwd_multi_node_test(api,
                      asic_type=asic_type)
 
 
+def sec_to_nanosec(secs):
```

Comment:
This function has been defined, implemented and used multiple times.

Reply:
Will move to a separate helper function